packages/core/src/lib/wallet/wallet.types.ts

```diff
@@ -113,6 +117,10 @@ export interface BrowserWalletSignAndSendTransactionsParams
 export type BrowserWalletBehaviour = Modify<
   BaseWalletBehaviour,
   {
+    importAccountsByEncryptedUrl?(
+      params: AccountImportEncryptedParams
+    ): Promise<void>;
```

Comment:
Just wondering, is `importAccountsByEncryptedUrl` need to be a promise? 
What I assumed was that this function should receive an argument(`AccountImportEncryptedParams`) and return string (url).